### PR TITLE
Update Matomo config docs

### PR DESCRIPTION
**Pull request by jazairi**
### Why these changes are being introduced:

While looking for internal docs on our Matomo implementation's privacy settings, I realized that much of the info in this doc is outdated.

### Relevant ticket(s):

https://mitlibraries.atlassian.net/browse/UXWS-1532

### How this addresses that need:

This adds more context around IP anonymization (the original task of UXWS-1532), removes some outdated info (mostly URLs), and removes redundant info that is either already documented in the UXWS wiki or the official Matomo docs.

### Side effects of this change:

We may want to document the IP anonymization setting on the UXWS wiki as well. I'll confirm this with UXWS.

### CODE REVIEW

File: misc/matomo.md [hunk 1]
```diff
@@ -10,35 +10,21 @@ the dockerfile and config is located on [GitHub](https://github.com/MITLibraries
...
-Add a new measurable,
-3. Give the measurable a useful name. We generally use the name the app is 
-known by internally, plus a note if it’s the staging app. E.g.: “Bento” 
-or “Bento (staging)”.
-4. Add the root URL for the application, as well as any aliases. It is 
-not necessary to add specific routes in the application.
-5. Leave the remaining settings as their defaults, then click ‘Save’.
-
-The website will now be listed under Websites > Manage.
+See [the official Matomo documentation](https://matomo.org/faq/how-to/create-and-manage-websites/). Note that only the
```
Comment (JPrevost) [inline misc/matomo.md]: Can you clarify this means what it says? Other accounts can access matomo but I suspect this is trying to convey that only superuser can do a specific thing?

Comment (jazairi) [reply misc/matomo.md]: 🤦 Good catch!

**Comment by jazairi** [comment]
@JPrevost I'm about to make a few additional changes here. Dhruti and I decided to move the Privacy section to the UXWS wiki. Just wanted to let you know that the re-review will include more than fixing that typo. 🙂

**Comment by jazairi** [comment]
@JPrevost could you take a look at the latest commit when you get the chance?

**Review by JPrevost — APPROVED** [review]

**Pull request merged**